src/components/organisms/header/header.html

```diff
@@ -0,0 +1,80 @@
+<header>
+  <div class="header">

```

Comment:
```suggestion
<header>
  <div class="header-inner">
```
`header` と `.header` が別々の要素として存在するのは紛らわしいので、より判別しやすい命名にできるとよりよいかと思います！
（上記は一例です）

セマンティック要素を使えている点は素晴らしいです 🙌